torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1258,15 +1258,7 @@ def sample_inputs_fliplr_flipud(op_info, device, dtype, requires_grad):
            supports_tensor_out=True,
            decorators=[skipCUDAIfNoMagma, skipCPUIfNoLapack],
            sample_inputs_func=sample_inputs_linalg_norm,
-           aten_name='linalg_norm',
```

Comment:
Now that pow() has been implemented for float16 & bfloat16 on CPU, SkipInfo for it should be removed.

Reply:
Looks like 3 other tests would have to be unskipped as well, as they're currently being skipped due to `pow()` not hitherto having been implemented for `bfloat16`.